### PR TITLE
Added user's score to Candy Command - Closes #947

### LINKED ISSUE #947: View Your Score in Candy Command
## Description
Currently, there is no way to view your (the member who invoked the command) score, I suggest adding another line to the resulting embed which tells the user their own candy score. 

## Would you like to implement this yourself?

- [ ] I'd like to implement this feature myself
- [X] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by evgriff**
## Relevant Issues

Closes #947 

## Description

Added user's score to Candy command by adding one additional function/variable, and including verbose to the candy commands message. Invoker of candy command will now see their candy score.
## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/utilities/issues.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ async def fetch_issues(
             log.trace(f"PR provided, querying GH pulls API for additional information: {pulls_url}")
             async with self.bot.http_session.get(pulls_url) as p:
                 pull_data = await p.json()
-                if pull_data["draft"]:
+                if "draft" in pull_data and pull_data["draft"]:
```
Comment (burakhanaksoy) [inline bot/exts/utilities/issues.py:142]: How about using ```pull_data.get("draft")``` instead? 

Same review as [979](https://github.com/python-discord/sir-lancebot/pull/979#pullrequestreview-829455026)

Comment (onerandomusername) [reply bot/exts/utilities/issues.py:142]: hm, this commit shouldn't even be on this pr.

@evgriff this change is out of scope of this pr, so you'll need to remove it. you can do it with either resetting your HEAD and then forcepushing, or making a commit which will revert this commit.

File: bot/exts/holidays/halloween/candy_collection.py [hunk 2]
```diff
@@ -173,6 +173,7 @@ async def send_no_candy_spook_message(
     async def candy(self, ctx: commands.Context) -> None:
         """Get the candy leaderboard and save to JSON."""
         records = await self.candy_records.items()
+        user = await self.bot.fetch_user(ctx.author.id)
```
Comment (Shivansh-007) [inline bot/exts/holidays/halloween/candy_collection.py:176]: We don't need to fetch the user here, we are using the user object to get the id/mention which can be done on `ctx.author` also, the two are the same objects.

**Comment by katimoth** [comment]
@ks129 If this code is good, could you approve this PR? @evgriff and I are doing this as a school assignment due tonight and need to have proof of a successful merge.

**Comment by HassanAbouelela** [comment]
@katimoth I agree with Shivnash's comments. Could you quickly make those changes?

**Comment by katimoth** [comment]
@HassanAbouelela Committed the suggestions

**Review by HassanAbouelela — APPROVED** [review]
Thanks for your work, and congrats on your first PR here 🎉

**Comment by katimoth** [comment]
> Thanks for your work, and congrats on your first PR here 🎉

Thank you !

**Review by D0rs4n — APPROVED** [review]
Tested out locally, works as expected.
Congrats on you first PR 🎉

**Pull request merged**